src/lang/terms/transitive.pl

```diff
@@ -100,3 +100,20 @@
     Goal=..[Functor,S,P,O]
   ),
   ask(Goal,Scope).
+
+
+     /*******************************
+     *        UNIT TESTS            *
+     *******************************/
+
+:- begin_tripledb_tests(
+    'lang_transitive',
+    'package://knowrob/owl/test/swrl.owl',
+    [ namespace('http://knowrob.org/kb/swrl_test#')
+    ]).
+
+test("ask ancestor of Lea") :-
+  assert_true(transitive(holds(test:'Lea', test:'hasAncestor', test:'Rex'))),
+  assert_false(transitive(holds(test:'Lea', test:'hasAncestor', test:'Ernest'))).
```

Comment:
I meant something different:

```Prolog
assert_false(holds(test:'Lea', test:'hasAncestor', test:'Ernest'))
```

Does that work?

Reply:
this works. Then, I did not get the proper context. What are the negative tests intended for?

Reply:
the one I referred to? Well that it does work with but not without transitive

Reply:
oh sorry, I meant:

```Prolog
assert_false(holds(test:'Lea', test:'hasAncestor', test:'Rex'))
```

Reply:
This works after removing transitive property for hasAncestor